### DISCUSSION TITLE
Add Reservation via Web UI

### DISCUSSION THREAD

**Discussion by DCMattyG** [Azure/ipam#56] [Ideas]
Hi @pcdiks, I saw you had opened and closed a ticket on this topic, so I wanted to understand better the request.

The reason that we haven't enabled the ability to create a reservation from the Web UI is that it would seem if a user intended to create a new vNET, it seems tedious to do this as a two-part action (create Reservation, then create vNET in Azure) as opposed to just creating the vNET.

I can imagine that the desire for this may be related to the fact that creating a new reservation reveals the next available CIDR, which otherwise you would need to manually derive. If this is the case, perhaps something like @letmetechyou suggested in #50 (Subnet planner/visualizer) may be useful?

Anyway, let me know your thoughts and I'm always happy to implement new functionality where it makes sense and is useful to other users.

Thanks!

**Comment by pcdiks** [comment]
No, I don't need it. I just wanted to do a quick test with the UI.

**Reply by DCMattyG** [reply]
Ok, no worries. Let me know if there's anything else I can be of assistance with!
